ui/src/geometry/geometry-model.ts

```diff
@@ -162,7 +162,6 @@ export type GeometryAlignment = {
     elements: GeometryElement[];
     profile?: GeometryProfile;
     cant?: GeometryCant;
-    trackNumberId: GeometryTrackNumberId;
```

Comment:
Vielä lopuksi frontin puolelta poistettu nuo redundantit kentät. Näitä ei käytetty missään.